src/dot.rs

```diff
@@ -109,40 +112,62 @@ pub enum Config {
     /// Do not print the graph/digraph string.
     GraphContentOnly,
     #[doc(hidden)]
-    _Incomplete(()),
+    // can never be constructed
+    _Incomplete(std::convert::Infallible),
```

Comment:
While interesting and something that has been considered before, having an inconstructible variant in some situations allows the user to omit it from match, which is sort of the opposite of what we want..